### PR TITLE
pulp-cli

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ehelms — APPROVED** [review]

File: packages/python-pulp-cli/python-pulp-cli.spec [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Requires:       python%{python3_pkgversion}-click < 9
+Requires:       python%{python3_pkgversion}-click >= 7.1.2
+Requires:       python%{python3_pkgversion}-click-shell < 3
+Requires:       python%{python3_pkgversion}-click-shell >= 2.1
+Requires:       python%{python3_pkgversion}-packaging
+Requires:       python%{python3_pkgversion}-pygments
+Requires:       python%{python3_pkgversion}-requests < 2.26
+Requires:       python%{python3_pkgversion}-requests >= 2.25.1
+Requires:       python%{python3_pkgversion}-setuptools
+Requires:       python%{python3_pkgversion}-toml = 0.10.2
```
Comment (ekohl) [inline packages/python-pulp-cli/python-pulp-cli.spec:35]: Why is this pinned on an exact version? Does upstream have a good reason for that? https://github.com/pulp/pulp-cli/commit/d3491ef903a4d166124f546faee326851d29ab47 doesn't really mention it.

Comment (evgeni) [reply packages/python-pulp-cli/python-pulp-cli.spec:35]: the lib doesn't offer a stable api (yet) :(

**Comment by evgeni** [comment]
```console
% koji add-pkg pulpcore-3.13-el8 python-pulp-cli python-click-shell --owner=kojiadm
Adding 2 packages to tag pulpcore-3.13-el8
% koji add-pkg pulpcore-3.13-el7 python-pulp-cli python-click-shell --owner=kojiadm
Adding 2 packages to tag pulpcore-3.13-el7
```

File: packages/python-pulp-cli/python-pulp-cli.spec (hunk 1 above)
Comment (ekohl) [reply packages/python-pulp-cli/python-pulp-cli.spec:35]: I guess that would have been too convenient for the ecosystem.

**Review by ekohl — APPROVED** [review]

**Pull request merged**